README.md

```diff
@@ -26,29 +36,43 @@ npm install cgroup-metrics
 
 ```javascript
 const cgroup = require('cgroup-metrics');
+
+// You can access each metric separately using the async functions for each metric
+
+// Memory Metrics
 const memory = cgroup.memory();
+const containerUsage = await memory.containerUsage();
+console.log(containerUsage);
+
+const containerUsagePercentage = await memory.containerUsagePercentage(containerUsage);
+console.log(containerUsagePercentage);
+
+// CPU Metrics
+const cpu = cgroup.cpu();
+const cpuacct_usage = await cpu.usage();
+console.log(`Total CPU usage: ${cpuacct_usage}`);
+
+const cpuacct_stats = await cpu.stat();
+console.log(`CPU user count: ${cpuacct_stat.user}`);
+console.log(`CPU system count: ${cpuacct_stat.system}`);
+
+const cpuacct_usage_percpu = await cpu.usage_percpu();
+console.log(`CPU usage per CPU task: ${cpuacct_usage_percpu}`);
+
+// Or you can use the function `getAllMetrics` to get an object of all the metrics
+const metrics = await cgroup.getAllMetrics();
```

Comment:
@jdelbick I would suggest to rename `cgroup.getAllMetrics()` to `cgroup.metrics()`.

This would be more consistent with the other method names (`stat()`, `usage()` ...).

Reply:
Good idea.